src/openms/source/DATASTRUCTURES/SuffixArraySeqan.cpp

```diff
@@ -171,7 +171,7 @@ namespace OpenMS
   //destructor
   SuffixArraySeqan::~SuffixArraySeqan()
   {
-
+    delete it_;
```

Comment:
hmm trashing seems a bit harsh as it is used by various other classes... maybe we should deprecate it first?